CMakeLists.txt

```diff
@@ -28,12 +29,17 @@ cmake_minimum_required(VERSION 3.13.1)
 #                        BOARD ${TFMBOARD}
 #                        CFGFILE "ConfigRegressionIPC"
 #                        OUT_VENEERS_FILE VENEERS_FILE
-#                        OUT_INCLUDE_PATH TFM_INCLUDE_PATH)
+#                        OUT_INCLUDE_PATH TFM_INCLUDE_PATH
+#                        BL2 True)
 function(trusted_firmware_build)
   set(options IPC)
-  set(oneValueArgs BINARY_DIR BOARD CFGFILE OUT_VENEERS_FILE OUT_INCLUDE_PATH)
+  set(oneValueArgs BINARY_DIR BOARD CFGFILE OUT_VENEERS_FILE OUT_INCLUDE_PATH BL2)
   cmake_parse_arguments(TFM "${options}" "${oneValueArgs}" "" ${ARGN})
 
+  if(NOT DEFINED TFM_BL2)
+    set(TFM_BL2 True) #by default True
+  endif()
```

Comment:
See other comment.
```suggestion
  if(DEFINED TFM_BL2)
    set(TFM_BL2_ARG "-DBL2=${TFM_BL2}")
  endif()
```

Reply:
I'm OK with this, but it will require updating the `mps2_an521_nonsecure` and `nucleo_l552ze_q` to set the BL2 status to true:

- https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/tfm_integration/tfm_ipc/CMakeLists.txt#L11
- https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/tfm_integration/psa_level_1/CMakeLists.txt#L10

Reply:
@microbuilder there is already BL2 set to True by default in Zephyr's TFM samples.